HST/product_labels/__init__.py

```
@@ -107,11 +103,12 @@ def label_hst_fits_filepaths(filepaths, root='', *,
         retrieval_date      date the file was retrieved from MAST, in yyyy-mm-dd format.
                             If blank, the date will be inferred from the file itself.
         logger              pdslogger to use; None for default EasyLogger.
-        replace_nans        True to rewrite each file without NaNs if NaNs are found.
         reset_dates         True to reset the modification date of each file to the date
                             found in the FITS header.
     """
 
+    replace_nans = False    # disabled feature
```

Flake8 (7.2.0)

[error] local variable 'replace_nans' is assigned to but never used

(F841)